### PR TITLE
DTGB-213: Fixed font size, line height and margins of headings.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/21-atoms/headings/heading-1/_heading-1.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@ h1,
 %h1 {
   @include theme('color', 'color-primary', 'heading-1-color');
 
-  margin-bottom: 3rem;
-  font-size: 2.625rem;
-  line-height: 3.5rem;
+  margin-bottom: 2.4rem;
+  font-size: 2.1rem;
+  line-height: 2.8rem;
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/headings/heading-1/_heading-1.scss]: ems not rems

File: styleguide/components/21-atoms/headings/heading-2/_heading-2.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@ h2,
 %h2 {
   @include theme('color', 'color-tertiary', 'heading-2-color');
 
-  margin-bottom: 3rem;
-  font-size: 2.25rem;
-  line-height: 3.375rem;
+  margin-bottom: 2.4rem;
+  font-size: 1.8rem;
+  line-height: 2.7rem;
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/headings/heading-2/_heading-2.scss]: ems not rems

File: styleguide/components/21-atoms/headings/heading-3/_heading-3.scss [hunk 3]
```diff
@@ -3,7 +3,7 @@ h3,
 %h3 {
   @include theme('color', 'color-tertiary', 'heading-3-color');
 
-  margin-bottom: 3rem;
-  font-size: 1.875rem;
-  line-height: 2.875rem;
+  margin-bottom: 2.4rem;
+  font-size: 1.5rem;
+  line-height: 2.3rem;
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/headings/heading-3/_heading-3.scss]: ems not rems

File: styleguide/components/21-atoms/headings/heading-4/_heading-4.scss [hunk 4]
```diff
@@ -3,7 +3,7 @@ h4,
 %h4 {
   @include theme('color', 'color-tertiary', 'heading-4-color');
 
-  margin-bottom: 3rem;
-  font-size: 1.5rem;
-  line-height: 2.375rem;
+  margin-bottom: 2.4rem;
+  font-size: 1.2rem;
+  line-height: 1.9rem;
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/headings/heading-4/_heading-4.scss]: ems not rems

File: styleguide/components/21-atoms/headings/heading-5/_heading-5.scss [hunk 5]
```diff
@@ -3,7 +3,7 @@ h5,
 %h5 {
   @include theme('color', 'color-tertiary', 'heading-5-color');
 
-  margin-bottom: 3rem;
-  font-size: 1.25rem;
-  line-height: 1.875rem;
+  margin-bottom: 2.4rem;
+  font-size: 1rem;
+  line-height: 1.5rem;
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/headings/heading-5/_heading-5.scss]: ems not rems

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/21-atoms/headings/heading-1/_heading-1.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/21-atoms/headings/heading-1/_heading-1.scss]: Fixed.

File: styleguide/components/21-atoms/headings/heading-2/_heading-2.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/21-atoms/headings/heading-2/_heading-2.scss]: Fixed.

File: styleguide/components/21-atoms/headings/heading-3/_heading-3.scss (hunk 3 above)
Comment (meirege) [reply styleguide/components/21-atoms/headings/heading-3/_heading-3.scss]: Fixed.

File: styleguide/components/21-atoms/headings/heading-4/_heading-4.scss (hunk 4 above)
Comment (meirege) [reply styleguide/components/21-atoms/headings/heading-4/_heading-4.scss]: Fixed.

File: styleguide/components/21-atoms/headings/heading-5/_heading-5.scss (hunk 5 above)
Comment (meirege) [reply styleguide/components/21-atoms/headings/heading-5/_heading-5.scss]: Fixed.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**